_docs_explore/22-security.md

```diff
@@ -72,3 +76,37 @@ It is possible to temporarily or permanently block access or delete identities,
 In addition, Enmeshed supports law enforcement up to a certain degree. As like any other central provider, it is possible to block access or delete content if the Backbone operator gets the official task to comply (although the Backbone operator wouldn't know who the user is or which content it deletes).
 
 Additionally, it is possible to point law enforcement organizations to other identities which have communicated with an unknown target identity in the past. If the actual real-world entity behind such an identity is known (e.g. a company) they might know who the target identity is and could further help law enforcement.
+
+# Setup Firewall
```

Comment:
Ich finde die ganzen neuen Chapters hier passen vom Informationsgehalt ggf. auch in Integrate?

Reply:
dann würden updates, virus scan, etc aber auch in Integrate passen

Reply:
Wir sollten mindestens im connector installation guide hierrauf verweisen.